Script/item.dat

```diff
@@ -5038,3 +5366,172 @@ ullrbone
+  MaterialConfigChances = { 6, 100, 100, 50, 50, 10, 5; }
+  Roundness = 10;
+  IsTwoHanded = true;
+  AttachedGod = SILVA;
+  WieldedBitmapPos = 160, 96;
+  EnchantmentPlusChance = 15;
+  DamageFlags = PIERCE;
+  Price = 500;
+  MinCharges = 2;
+  MaxCharges = 8;
+  IsKamikazeWeapon = true;
+  TeleportPriority = 1000;
+
+  Config BROKEN;
+  {
+    Possibility = 0;
+    BitmapPos = 112, 352;
+    WieldedBitmapPos = 176, 96;
+    MainMaterialConfig = { 5, GREEN_CRYSTAL, SUN_CRYSTAL, SAPPHIRE, RUBY, DIAMOND; }
+    MaterialConfigChances = { 5, 100, 100, 50, 50, 5; }
+    EnchantmentPlusChance = 30;
+  }
+}
+
+cauldron /* materialcontainer-> */
+{
+  Possibility = 0;
+  Category = POTION;
+  StrengthModifier = 350;
+  DefaultSize = 150;
+  DefaultMainVolume = 80000;
+  DefaultSecondaryVolume = 25000;
+  BitmapPos = 80, 304;
+  NameSingular = "cauldron";
+  MainMaterialConfig == COPPER;
+  SecondaryMaterialConfig = { 10, WATER, HEALING_LIQUID, POISON_LIQUID, VODKA, TROLL_BLOOD, LIQUID_DARKNESS, YELLOW_SLIME, SWEAT, SICK_BLOOD, LIGHT_FROG_BLOOD; }
+  MaterialConfigChances = { 10, 75, 50, 25, 50, 25, 5, 5, 1, 5, 25; }
+  Roundness = 100;
+  CanBeBroken = false;
+  AttachedGod = NONE;
+  WieldedBitmapPos = 176, 144;
+  IsValuable = false;
+  CanBeGeneratedInContainer = false;
+  CanBePickedUp = false;
+}
+
+trinket
+{
+  Category = MISC;
+  IsAbstract = true;
+  CanBeBroken = false;
+  CanBeBurned = false;
+  HasSecondaryMaterial = true;
+  Possibility = 10;
+  IsValuable = false;
+  WieldedBitmapPos = 176, 144;
+
+  Config POTTED_CACTUS;
+  {
+    DefaultSize = 30;
+    DefaultMainVolume = 500;
+    DefaultSecondaryVolume = 50;
+    BitmapPos = 64, 320;
+    FormModifier = 20;
+    StrengthModifier = 50;
+    Adjective = "potted";
+    NameSingular = "cactus";
+    MainMaterialConfig == MARBLE;
+    SecondaryMaterialConfig == PLANT_FIBER;
+    Roundness = 60;
+    AttachedGod = SCABIES;
+    IsSadistWeapon = true;
+    DamageFlags = PIERCE;
+  }
+
+  Config POTTED_PLANT;
+  {
+    DefaultSize = 30;
+    DefaultMainVolume = 500;
+    DefaultSecondaryVolume = 50;
+    BitmapPos = 112, 336;
+    FormModifier = 20;
+    StrengthModifier = 50;
+    Adjective = "potted";
+    NameSingular = "plant";
+    MainMaterialConfig == MARBLE;
+    SecondaryMaterialConfig == PLANT_FIBER;
+    Roundness = 60;
+    AttachedGod = SILVA;
+  }
+
+  Config DEAD_FISH;
+  {
+    DefaultSize = 30;
+    DefaultMainVolume = 250;
+    HasSecondaryMaterial = false;
+    BitmapPos = 80, 336;
+    FormModifier = 25;
+    StrengthModifier = 75;
+    Adjective = "dead";
+    NameSingular = "fish";
+    MainMaterialConfig == DOLPHIN_FLESH; /* maybe change */
+    Roundness = 15;
+    AttachedGod = SEGES;
+  }
+
+  Config BONE_FISH;
+  {
+    DefaultSize = 30;
+    DefaultMainVolume = 150;
+    HasSecondaryMaterial = false;
+    BitmapPos = 96, 336;
+    FormModifier = 25;
+    StrengthModifier = 75;
+    Adjective = "fish";
+    NameSingular = "bones";
+    NamePlural = "bones";
```

Comment:
Minor issue, it will say things like "A fish bones is lying here"

Ideally "Some fish bones are lying here" or the name would have to be changed to something singular like "fish skeleton"